tests/test_addDistortionModel.py

```diff
@@ -0,0 +1,124 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import lsst.utils.tests
+import lsst.afw.geom as afwGeom
+from lsst.afw.coord import IcrsCoord
+from lsst.afw.image import ExposureF, ExposureInfo, Calib, VisitInfo
+from lsst.afw.geom import makeCdMatrix, makeSkyWcs
+from lsst.afw.geom.wcsUtils import makeDistortedTanWcs
+from lsst.afw.cameraGeom import FIELD_ANGLE, FOCAL_PLANE, PIXELS
+from lsst.afw.cameraGeom.testUtils import DetectorWrapper, CameraWrapper
+from lsst.afw.geom.utils import wcsAlmostEqualOverBBox
+from lsst.ip.isr import IsrTask
+
+
+class ApplyLookupTableTestCase(lsst.utils.tests.TestCase):
+    """Test IsrTask.addDistortionModel
+    """
+
+    def setUp(self):
+        self.camera = CameraWrapper().camera
+        self.detector = DetectorWrapper().detector
+        self.crpix = afwGeom.Point2D(50, 100)
+        self.crval = IcrsCoord(36 * afwGeom.degrees, 71 * afwGeom.degrees)
+        scale = 1.0 * afwGeom.arcseconds
```

Comment:
https://developer.lsst.io/coding/python_style_guide.html#binary-operators-should-be-surrounded-by-a-single-space-except-for